bin/load-mocks

```diff
@@ -264,10 +264,15 @@ def main(num_events=1):
                 }
             )
 
-            release = Release.objects.get_or_create(
+            release, created = Release.objects.get_or_create(
                 version=sha1(uuid4().bytes).hexdigest(),
                 project=project,
-            )[0]
+            )
+
+            if created:
+                release.organization_id = project.organization_id
```

Comment:
you can use defaults={'organization_id': project.organization_id} in the get_or_create call